maths/shreedharacharya.dart

```diff
@@ -0,0 +1,26 @@
+import 'dart:math';
+//this function return a list of roots of a quadratic equation
+// [x1, x2] where x1 and x2 are roots of
+// aX^2 + bX + c = 0
+List shreedharacharya(double a,double b,double c) 
+{
+  double d = b*b - 4*a*c;
+  List A=[0,0];
+  if(d<0)
```

Comment:
You could also cover edge case when `d == 0`. There will be just one real root in this case.

Reply:
Sir that will be included in d>0

Reply:
Yes, but right now when `d==0` you are appending list with 2 real roots, when there is just one. 
`(-b - sqrt(d))/(2*a)` and `(-b + sqrt(d))/(2*a)` will both simplify to `-b /(2 * a)`

Reply:
I request you to suggest me the change.